src/main/resources/templates/Index.html

```diff
@@ -1,85 +1,87 @@
 <!DOCTYPE html>
-<html xml:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
 <head>
-  <link rel="stylesheet" href="index.css">
+  <title>Voluntier</title>
+  <link rel="stylesheet" th:href="@{css/Index.css}">
+
   <script>
     function openForm() {
       document.getElementById("myForm1").style.display = "block";
     }
 
-    function closeForm() {
-      document.getElementById("myForm").style.display = "none";
+    function closeForm1() {
+      document.getElementById("myForm1").style.display = "none";
+    }
+
+    function closeForm2() {
+        document.getElementById("myForm2").style.display = "none";
     }
   </script>
 </head>
 <body>
 <div class="content">
   <header>
-    <img src="images.png" alt="Logo">
-
+    <img th:src="@{/images/images.png}" alt="Logo">
   </header>
+
   <div class="tab">
-    <button><a href="index.html">About this App</a></button>
+    <button><a href="/">About this App</a></button>
     <button><a href="#">Account</a></button>
     <button><a href="#">Password</a></button>
-    
-    <button><a href="help.html">Help</a></button>
-    <button><a href="#">Apply</a></button>
-    <button><a href="post.html">Create/Post</a></button>
 
+    <button><a href="help">Help</a></button>
+    <button><a href="#">Apply</a></button>
+    <button><a href="post">Create/Post</a></button>
   </div>
 
   <div class="login">
     <button class="open-button" onclick="openForm()">Log In</button>
-
-
-    <p th:text="${volunteer}">
-
-
+    <p th:text="${volunteer}"></p>
```

Comment:
Adding end `</p>` tag for readability and to ensure HTML compliance in the future. While in this case you may omit the closing tag since it's followed immediately by a `<div>` tag, I would recommend closing it in case you add a tag in the future that does not allow you to omit the end `</p>` tag. If you want to stick with the current behavior, I recommend looking at the reference to determine when the end `</p>` tag is necessary. [Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p)